<launch>

  <!-- Human-aware Plugin for MoveIt! -->
  <arg name="planning_plugin" value="ha_planner/DgacoPlannerManager" />

<!--  <arg name="planning_plugin" value="ompl_interface/OMPLPlanner" />-->

<!--  <arg name="planning_adapters" value="default_planner_request_adapters/AddTimeParameterization" />-->

<!-- The request adapters (plugins) used when planning with OMPL.
     ORDER MATTERS -->
<arg name="start_state_max_bounds_error" value="0.1" />

<param name="planning_plugin" value="$(arg planning_plugin)" />

<rosparam command="load" file="$(find dimostratore_moveit_config)/config/ha_planning.yaml"/>
<rosparam command="load" file="$(find inbound_pick)/config/config.yaml"/>
<node pkg="inbound_pick" type="inbound_pick_server" name="inbound_pick_server" />
<node pkg="inbound_pick" type="inbound_pick_loader" name="inbound_pick_loader" />
</launch>


